src/NServiceBus.Metrics.ServiceControl.Tests/TagExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+
+static class TagExtensions
```

Comment:
This is not needed.

Reply:
It is. At least R# and compiler says so :P It's used in asserts like one below:

```c#
   var counterBasedKey = tags.GetTagValue("key");
```

Reply:
@Scooletz don't argue with @tmasternak . Just fix it and then argue with the compiler. It will be easier :P